administrator/components/com_patchtester/PatchTester/Model/PullsModel.php

```diff
@@ -345,8 +345,8 @@ public function requestFromGithub($page)
 			// Build the data object to store in the database
 			$pullData = array(
 				$pull->number,
-				$this->getDb()->quote($pull->title),
-				$this->getDb()->quote(\JHtml::_('string.truncate', $pull->body, 100)),
+				$this->getDb()->quote(\JHtml::_('string.truncate', $pull->title, 200)),
+				$this->getDb()->quote(\JHtml::_('string.truncate', $pull->body, 150)),
```

Comment:
Does truncate always cap out at the specified character limit?  This was using truncateComplex before with the shorter limit since it was supposed to handle logical cutoffs (but still left too long a description).